gui/default/syncthing/core/syncthingController.js

```diff
@@ -1402,6 +1402,11 @@ angular.module('syncthing.core')
                 autoNormalize: true
             };
             $scope.editingExisting = false;
+            $scope.newIgnores = {
+                edited: false,
```

Comment:
It can exist even if we cleaned up: I exclusively set ignores by editing the .stignore file directly and I always create it before adding the folder to prevent unwanted files making it into the index.
And setIgnores is called at the end on saving the new folder, then it overwrites the file on disk and only then is getignores called to reply to the REST call with the new config. The difficulty in this PR was always that you can't populate what is on disk as the folder does not yet exist and the path is not fixed yet. Otherwise we wouldn't even be talking about overwriting.